src/compiler/parser.ts

```diff
@@ -436,6 +436,13 @@ module ts {
         // attached to the EOF token.
         let parseErrorBeforeNextFinishedNode: boolean = false;
 
+        export const enum StatementFlags {

```

Comment:
We're going around in circles. I don't want to translate the flags from one form to another. It adds nothing but noise and opportunity for error. We could potentially get rid of StatementFlags and use (a subset of) ParsingContext for the same purpose. Then we'd be comparing apples to apples. Otherwise, I don't see any point in changing this.